start.sh

```diff
@@ -81,11 +81,14 @@ if [ "${IS_CRON_POD:-"false",,}" == "false" ]; then
         echo "Starting Gunicorn for PTVSD debugging"
         # Workers need to be set to 1 for PTVSD
         GUNICORN_WORKERS=1
+        GUNICORN_RELOAD="--reload"
+        GUNICORN_TIMEOUT=0
```

Comment:
I can see `"PTVSD_ENABLE": true,` stopping on breakpoint is happening and do not  disconnect after 30 sec.